### PR TITLE
add OpenBSD support

### PR DESCRIPTION
**Pull request by Ilya33**
Hello.

Add #cgo directives for OpenBSD

Best Regards, Ilya Pavlov

### CODE REVIEW

File: driver_openal.go [hunk 1]
```diff
@@ -12,14 +12,16 @@
...
+// +build darwin freebsd openbsd
 // +build !js
 // +build !android
 
 package oto
 
 // #cgo darwin  LDFLAGS: -framework OpenAL
 // #cgo freebsd LDFLAGS: -lopenal
+// #cgo openbsd LDFLAGS: -L/usr/local/lib/ -lopenal
+// #cgo openbsd CFLAGS: -I/usr/local/include/
```
Comment (hajimehoshi) [inline driver_openal.go]: Is adding `/usr/local/lib` mandatory? On FreeBSD, I expect users to add such values to environment variables.

**Comment by Ilya33** [comment]
On OpenBSD non-system (and packages) headers are located in /usr/local/include/.
So user may run `CGO_CFLAGS="-I/usr/local/include/"  CGO_LDFLAGS="-L/usr/local/lib/" go build` for build the oto.

**Comment by hajimehoshi** [comment]
Hmm, can we use `pkg-config` instead of hard coding paths?

https://golang.org/cmd/cgo/#hdr-Using_cgo_with_the_go_command

**Comment by Ilya33** [comment]
fixed in [#PR71](https://github.com/hajimehoshi/oto/pull/71)

**Pull request merged**